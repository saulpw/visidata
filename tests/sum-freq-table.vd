sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	Units		type-int		#	set type of current column to int
sample	Units		aggregate-col	sum	+	Add aggregator to current column
sample	Units		freq-col		Shift+F	open Frequency Table grouped on current column, with aggregations of other columns
sample_Units_freq	Units		sort-asc		[	sort ascending by current column; replace any existing sort criteria
