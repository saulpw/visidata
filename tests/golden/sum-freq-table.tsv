Units	count	Units_sum
2	1	2
3	1	3
4	1	4
5	1	5
7	2	14
11	1	11
14	1	14
15	1	15
16	1	16
27	1	27
28	2	56
29	1	29
32	1	32
35	1	35
36	1	36
42	1	42
46	1	46
50	2	100
53	1	53
55	1	55
56	1	56
57	1	57
60	2	120
62	1	62
64	1	64
66	1	66
67	1	67
74	1	74
75	1	75
76	1	76
80	1	80
81	1	81
87	1	87
90	2	180
94	1	94
95	1	95
96	2	192
